web_tool_script_2.R

```diff
@@ -200,3 +197,6 @@ if (file.exists(bonds_inputs_file)){
   }
 }
 
+invisible(set_portfolio_parameters(file_path = paste0(par_file_path,"/",portfolio_name_ref_all,"_PortfolioParameters.yml")))
+
+source(paste0(stress_test_path, "web_tool_stress_test.R"))
```

Comment:
Paths are more robust and portable if you create them with dedicated functions such as `fs::path()` and `file.path()`. This code base is contamined with paths created with `paste()`; I recommend to fix this as you touch the code. That way, at least the most commonly visited lines will be eventually refactored.

Reply:
good point, I will make that a prio whenever I touch a path